### PR TITLE
NuGet pack append prerelease suffix to version

### PR DESCRIPTION
**Pull request by logankp**
Added a -Suffix option to the NuGet pack command.
- Example: nuget pack proj1.csproj -Suffix alpha
#694

### CODE REVIEW

File: src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx [hunk 1]
```diff
@@ -5302,4 +5302,7 @@ nuget locals global-packages -list</value>
   <data name="LocalsCommandListDescription" xml:space="preserve">
     <value>List the selected local resources or cache location(s).</value>
   </data>
+  <data name="PackageCommandSuffixDescription" xml:space="preserve">
+    <value>Adds a suffix to the generated version</value>
```
Comment (yishaigalatzer) [inline src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx]: dot at the end of the sentence, and try to be more descriptive, this reads like the result version will be 1.1.0-something 123-SUFFIX

File: test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPackCommandTest.cs [hunk 2]
```diff
@@ -1854,6 +1854,37 @@ public string F2(int a)
             }
         }
 
+        [Fact]
+        public void PackCommand_VersionSuffixIsAssigned()
+        {
+            var nugetexe = Util.GetNuGetExePath();
+            var workingDirectory = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString());
```
Comment (yishaigalatzer) [inline test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPackCommandTest.cs]: Don't use path.GetTempPath, see the approach in the follwing PR - https://github.com/NuGet/NuGet.Client/pull/122/files

**Comment by logankp** [comment]
This has been updated with the suggested fixes (not sure if updates send notifications or not).

**Comment by yishaigalatzer** [comment]
@feiling this looks good to me, can you take another look, squash and merge?

**Comment by feiling** [comment]
Looks good. I'll merge the PR.

**Comment by feiling** [comment]
PR merged. https://github.com/NuGet/NuGet.Client/commit/afc65ae26b6f79b85f7a8deddab18c2cc3182b70

**Pull request closed without merging**